src/amo/reducers/abuse.js

```diff
@@ -155,6 +155,7 @@ type Action =
   | ShowAddonAbuseReportUIAction;
 
 export default function abuseReducer(
+  // eslint-disable-next-line default-param-last
```

Comment:
Similar to my comment about the earlier instance of disabling this rule, could we alternatively add a default value for `action`, say of `''`? Or do you think that would be a worse offense?

Reply:
I've played with this, see resolved thread over https://github.com/mozilla/addons-frontend/pull/11940#pullrequestreview-1211512111 - Having to define a default action and make flow happy about it was a pain, especially as it's not going to be used anywhere (so we'd also ideally need to add an `invariant` or something)